docs/running-from-github.md

```diff
@@ -0,0 +1,43 @@
+# Running this test suite from Github Actions
+
+- Go to the [workflow].
+- Click on 'Run Workflow'
+- Fill in the values as needed. (See [options](#options))
+- Click on the second (green) "Run Workflow" button at the bottom of the form
+  
+
+## Options
+
+### `target-idp-env`
+
+Options are `eval` (default) and `prod`. This is the IdP you will be testing.
+
```

Comment:
and if this field is left blank, it uses prod. Unless something has changed in that regard.

Reply:
No, it uses eval by default in the workflow. And it will be filled in as such always.